website/docs/reference/widgets/input.md

```diff
@@ -1,258 +1,118 @@
-* Left - It aligns the text to the left of the input box. When you select **Left** alignment, you get additional settings that you can use to control the alignment and define the text's width.
-  * Alignment - With the help of alignment, you can define the placement of the text in accordance with the position of the input box. You can choose:
-    * Left - It aligns the text to the widget's left boundary that's away from the input box.
-    * Right - It aligns the text closer to the input box.
-  * Width - With the help of width, you can define the **number of columns** in the **grid** that surrounds the widget. You can specify how close or far the text can be placed to the input box.
-* Auto - It automatically adjusts the position of the text based on the input box's height.
-
-:::info
-Columns are the dashed lines (-----) that surround a widget when you try to drag, and drop it on the canvas.
-:::
+## Input validation
 
-<VideoEmbed host="youtube" videoId="7VFebef9JZg" title="How to set label properties?" caption="How to set label properties?"/>
+Input widgets gather user-provided data, which can be accessed on the widget's `text` property. However, to prevent polluting your dataset with messy or incorrectly formatted data, it's important to guide users toward providing input in a usable format.
 
-#### Placeholder
+### User instruction
 
-You can set a proxy text/value inside the input box using the `placeholder` property. It can be any message or hint for the expected input.
+The Input widget has useful properties for communicating expected data formats and boundaries to users. Providing instructions and expectations to the user up-front helps improve their responses and reduces the chance of messy data.
 
-<VideoEmbed host="youtube" videoId="576Bfo8htf0" title="Placeholder" caption="Placeholder"/>
+When you create your Input widget, give it a **Placeholder** value that shows the user an example of your expected response. These values are shown in grey when the widget loads and disappear instantly when the user starts typing.
 
-#### Default text
+You can also provide a **Tooltip** filled with a more detailed outline of what you want. This is especially useful if your input field has special rules or requirements that aren't obvious, such as requirements for building a strong password.
 
-This property lets you set a default value for the widget before the user has made any changes.
+<figure>
+  <img src="/img/input-tooltip-placeholder.png" style={{width:"100%", height:"auto"}} alt="Use Tooltips and Placeholders to show users how to format their input."/>
+  <figcaption align = "center"><i>Use Tooltips and Placeholders to show users how to format their input.</i></figcaption>
+</figure>
 
-One good example of using this property is to bind it with a column of the selected row of a table. `Default text` fills the input widget with the value at that table's cell.
+### Validation properties
```

Comment:
don't think we need this as. subheading . directly explain everything under Data Validation heading